src/library/optional_password_vsd_gui.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import importlib
+from ansible.module_utils.basic import AnsibleModule
+
+VSPK = None
+
+DOCUMENTATION = '''
+
+---
+module: optional_password_vsd_gui
+short_description:  Adding an option to provide an optional VSD GUI password
```

Comment:
Adding an option to change vsd gui password.

Filename should also change to change_vsd_gui_password.py so that if someone is looking for it, it's easier to find.